WebM: Compilation fail on Linux/ARM

**Operating system or device - Godot version:**
Arch Linux ARM

**Issue description** (what happened, and what was expected):
Cannot link on Arch Linux ARM (missing `vpx_ports/arm_cpudetect.c` in compilation source files):
```
modules/libmodules.x11.opt.tools.32.a(systemdependent.x11.opt.tools.32.o): In function `vp8_machine_specific_config':
thirdparty/libvpx/vp8/common/generic/systemdependent.c:(.text+0x28): undefined reference to `arm_cpu_caps'
```
Also ARM assembly code is not used for Linux/ARM architecture.

--- a/modules/webm/libvpx/SCsub
+++ b/modules/webm/libvpx/SCsub
@@ -263,6 +263,8 @@ if env["platform"] == 'uwp':
     else:
         webm_cpu_x86 = True
 else:
+    import platform
+    is_x11_or_server_arm = ((env["platform"] == 'x11' or env["platform"] == 'server') and platform.machine().startswith('arm'))
     is_ios_x86 = (env["platform"] == 'iphone' and env["ios_sim"] == "yes")
     is_android_x86 = (env["platform"] == 'android' and env["android_arch"] == 'x86')
     if is_android_x86:
@@ -270,8 +272,8 @@ else:
     if osx_fat:
         webm_cpu_x86 = True
     else:
-        webm_cpu_x86 = (cpu_bits == '32' or cpu_bits == '64') and (env["platform"] == 'windows' or env["platform"] == 'x11' or env["platform"] == 'osx' or env["platform"] == 'haiku' or is_android_x86 or is_ios_x86)
-        webm_cpu_arm = (not is_ios_x86 and env["platform"] == 'iphone') or env["platform"] == 'bb10' or (not is_android_x86 and env["platform"] == 'android')
+        webm_cpu_x86 = not is_x11_or_server_arm and (cpu_bits == '32' or cpu_bits == '64') and (env["platform"] == 'windows' or env["platform"] == 'x11' or env["platform"] == 'osx' or env["platform"] == 'haiku' or is_android_x86 or is_ios_x86)
+        webm_cpu_arm = is_x11_or_server_arm or (not is_ios_x86 and env["platform"] == 'iphone') or env["platform"] == 'bb10' or (not is_android_x86 and env["platform"] == 'android')
 
 if webm_cpu_x86:
     import subprocess
@@ -331,7 +333,7 @@ if webm_cpu_x86:
 if webm_cpu_arm:
     if env["platform"] == 'iphone':
         env_libvpx["ASFLAGS"] = '-arch armv7'
-    elif env["platform"] == 'android':
+    elif env["platform"] == 'android' or env["platform"] == 'x11' or env["platform"] == 'server':
         env_libvpx["ASFLAGS"] = '-mfpu=neon'
     elif env["platform"] == 'uwp':
         env_libvpx["AS"] = 'armasm'

